goplugin/file_loader.go

```diff
@@ -9,19 +9,15 @@ import (
 
 var log = logger.Get()
 
-func init() {
-	pluginStorage = FileSystemStorage{}
-}
-
-// interface so we can test which plugin should be loaded
-type storage interface {
-	fileExist(string) bool
+// Storage interface so we can test which plugin should be loaded
+type Storage interface {
```

Comment:
I think this interface could stay private, we only need to implement it for the package (not outside)